### PR TITLE
[CS-35313] Live preview duplicate data

### PR DESCRIPTION
**Pull request by uttamukkoji**

### CODE REVIEW

File: Contentstack.Core/Internals/HttpRequestHandler.cs
Comment (ishaileshmishra) [inline Contentstack.Core/Internals/HttpRequestHandler.cs:51]: Can we put the **version** in config/constant file instead of making changes here

File: SECURITY.md
Comment (ishaileshmishra) [inline SECURITY.md:3]: thanks for adding SECURITY.md :)

**Review by aravindbuilt — APPROVED** [review]

**Review by nandeesh-gajula — APPROVED** [review]
Approved on behalf of sdk after security review

**Pull request merged**
